src/containers/profile/__test__/profile.test.js

```diff
@@ -0,0 +1,22 @@
+// React Libraries
+import React from 'react';
```

Comment:
`react-redux` is not part of `React` libraries
it should be under, third-party libraries

Reply:
done